v2-migration-tool/src/main/java/software/amazon/awssdk/v2migration/NewClassToBuilder.java

```diff
@@ -13,9 +13,9 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.migration.internal.recipe;
+package software.amazon.awssdk.v2migration;
```

Comment:
I moved them out of internal package even though they are internal APIs to make them consistent with other recipes and follow the recipe convention https://docs.openrewrite.org/authoring-recipes/recipe-conventions-and-best-practices

Also, they show up in the logs.
```
Changes have been made to test/v2-migration-tool-tests/target/generated-test-sources/gradle/project/build.gradle by:
    software.amazon.awssdk.v2migration.AwsSdkForJavaV1ToV2
        software.amazon.awssdk.v2migration.UpgradeSdkDependencies
            org.openrewrite.java.dependencies.ChangeDependency: {oldGroupId=com.amazonaws, oldArtifactId=aws-java-sdk-dynamodb, newGroupId=software.amazon.awssdk, newArtifactId=dynamodb, newVersion=2.26.16-SNAPSHOT}

```